ai-agent/api/routers/job_search.py

```
@@ -0,0 +1,26 @@
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)